types/stream-meter/index.d.ts

```diff
@@ -1,15 +1,46 @@
 /// <reference types="node" />
 
-import { Transform } from "stream";
-
-declare function m(maxBytes?: number): m.StreamMeter;
+import { Transform } from "node:stream";
 
 declare namespace m {
-    export class StreamMeter extends Transform {
-        constructor(maxBytes?: number);
+    interface StreamMeterConstruct {
```

Comment:
I don't understand why this is needed; is it just to make it constructable without new? Reading the package's source, I don't see why it wouldn't work with `new`.

Reply:
The original typedef is 

```ts
declare function m(maxBytes?: number): m.StreamMeter;
```

which does not work when you want to call it with `new()`. 

But by doing interface-style, the function can be recognized by tsc as both factory func and class constructor.

See a [ts playground demo](https://tsplay.dev/mx7pxN).

Reply:
I guess I had it backwardsish.